.gitlab-ci.yml

```diff
@@ -379,7 +379,6 @@ OSTree simplified installer:
           - rhos-01/rhel-8.7-nightly-x86_64
           - rhos-01/centos-stream-8-x86_64
           - rhos-01/rhel-9.1-nightly-x86_64
-          - rhos-01/centos-stream-9-x86_64
```

Comment:
I would prefer commenting this line out and adding a note with the BZs. Just so it haunts us until we solve the issue.